test/functionalTest/cases/719_servicepath_for_registrations/cm_forwards_cb_with_servicePath.test

```diff
@@ -0,0 +1,456 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I'm not so sure 15 will change, the initial service -path for 14 would have, so I changed it (first service path is not found).
However, these things will change and if a functest becomes invalid because the broker is *better*, that is just OK.